### ISSUE TITLE
Permission for Reusage?

### ISSUE THREAD

**Issue by diveyez** [overfl0/Bulletproof-Arma-Launcher#284]
Lukasz, I am Ricky

Im pulling your work to my Bitbucket and Gitlab to deploy it into my VM's and see if it needs any pull work done on the repo, however I am wondering if it is possible to use this as the way people start the Arma Binary with additions to params and userconfig scripts?

The reasoning is, using this will let me script the entire setup so they don't have to do any setup and can just dive into an automatically dynamic enhanced populated environment.

I have been putting the HC's into Docker's and scripting them separately. This makes for an amazing experience. If you would be willing to issue permissions, I can Milestone that on my Jira and send you a link to join up on BitBucket.

I realize you have not worked on this in a while, and yes it is open sources. But I really like this work. Smooth brother.

**Comment by overfl0** [comment]
Hey!

I'm not 100% sure if I follow what you want to do, but as per the GPLv3 license this project is using (see LICENSE file), you're free to take, modify and reuse the code as long as your modified version is also GPLv3-licensed and you give credit to the original creators (basically me and @Sighter).

Regarding Github/Gitlab/Bitbucket stuff (I've seen your repository), I'd prefer to not spread this project over several hosting services, meaning you're free to do whatever you want with the code, of course: fork it, develop your fork on other services, etc..., but I would prefer the original version (and any PRs contributed back to the project) to be submitted here, on Github.

If you're talking about command-line switches for the program, they are already working and implemented (I'm using them for the autoupdate mechanism, for example) but depending on what you actually want to do (that's the part I don't fully understand) maybe it will be easier for you to just pre-generate a settings file that will contain the presets you want people to have, instead of adding additional switches.

It's true that I've not worked on this for a while and some of the _Bulletproof Environment_ stuff is probably not available or straight out not working anymore. If I had to write that part from scratch, I would not create any VM this time but just keep it simple by using a virtualenv, instead. I wasn't that knowledgeable in that area, at that time.

Btw I have a py3 branch where I started porting the whole launcher to Python 3 but put that on hold due to lack of time. I may revisit that at a later time.
